types/express-brute-redis/index.d.ts

```diff
@@ -0,0 +1,52 @@
+// Type definitions for express-brute-redis 0.0
+// Project: https://github.com/AdamPflug/express-brute-redis
+// Definitions by: Scott Harwell <https://github.com/scottharwell>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { ClientOpts } from "redis";
+
+/**
+ * @summary Redis store for Express Brute
+ * @class
+ */
+declare class express_brute_redis {
+    /**
+     * Default options for the Redis client
+     */
+    static defaults: ClientOpts;
+
+    /**
+     * @summary constructor
+     * @param options Options to configure the Redis client.
+     */
+    constructor(options?: ClientOpts, ...args: any[]);
+
+    /**
+     * @summary Sets a key in Redis storage.
+     * @param key
+     * @param value
+     * @param lifetime
+     * @param callback
+     */
+    set(key: string, value: string, lifetime?: number, callback?: SetCallback): void;
+
+    /**
+     * @summary Gets a key in Redis storage.
+     * @param key
+     * @param callback
+     */
+    get(key: string, callback?: GetCallback): void;
+
+    /**
+     * @summary Resets a key in Redis storage.
+     * @param key
+     * @param callback
+     */
+    reset(key: string, callback?: ResetCallback, ...args: any[]): void;
+}
+
+type SetCallback = (sender?: express_brute_redis) => void;
+type GetCallback = (err?: Error, data?: any) => void;
+type ResetCallback = (err?: Error, data?: any) => void;
```

Comment:
A couple comments:
- Please don't use optional parameters in callbacks, as discussed here: http://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html
- Please either inline these types (since you only use each of them once) or export them as discussed at the bottom of the template: http://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-class-d-ts.html.